### ISSUE TITLE
changing system run state via web console is not possible

### ISSUE THREAD

**Issue by christhephish** [ZoneMinder/zoneminder#2490]
**Describe Your Environment**
- Zoneminder 1.32.3
- apt via debian experimental repos (sid)
- Debian 9 (jessie - fully updated)

**If the issue concerns a camera**
- Issue does not DIRECTLY concern a camera, but the underlying system that monitors cameras
- frame rate n/a
- resolution n/a
- ZoneMinder Source Type: .deb?

**Describe the bug**
Attempting to change system status via API is nonfunctional. Button on nav-bar does not update, dialog that pops up upon click has no apparent function, commands ARE logged @ debug level when attempting to change system state...yet they do not work.

**To Reproduce**
1. Go to nav bar on web API 
2. Click on (RUNNING) or (STOPPED)
3. Utilize subsequent popup window regarding "RUN STATE"
4. All attempts at controlling system run-state via subsequent UI window result in no change in system status.

**Expected behavior**
Attemping to control the system run state via the web API are futile.  Debug level logging shows API commands being logged, yet system does not respond appropriately to START, RESTART or STOP commands via API window.  (Don't know what default option is there, but it does nothing).  Monitoring log in realtime shows commands, yet system apparently ignores them.  Monitoring logs in realtime whilst issuing commands via systemd result in rapid system state changes within seconds.  Web API does update status when instructed via command line.

**Debug Logs**
```

2019-01-31 09:38:16.225840 zmdc[30046].INF-zmdc.pl ['zmwatch.pl' exited, signal 14]
2019-01-31 09:38:16.178210 zmdc[30046].INF-zmdc.pl ['zmc -m 1' exited normally]
2019-01-31 09:38:16.084010 zmdc[30046].INF-zmdc.pl ['zmstats.pl' exited, signal 14]
2019-01-31 09:38:15.985970 zmdc[30046].INF-zmdc.pl ['zmfilter.pl --filter_id=2 --daemon' exited normally]
2019-01-31 09:38:15.928353 zmc_m2[30074].INF-zm_signal.cpp/40 [Got signal 15 (Terminated), exiting]
2019-01-31 09:38:15.928300 zmdc[30046].INF-zmdc.pl ['zmfilter.pl --filter_id=1 --daemon' exited normally]
2019-01-31 09:38:15.905075 zmc_m3[30077].INF-zm_signal.cpp/40 [Got signal 15 (Terminated), exiting]
2019-01-31 09:38:15.905010 zmdc[30046].INF-zmdc.pl ['zmc -m 2' sending stop to pid 30074 at 19/01/31 09:38:15]
2019-01-31 09:38:15.882240 zmdc[30046].INF-zmdc.pl ['zmc -m 3' sending stop to pid 30077 at 19/01/31 09:38:15]
2019-01-31 09:38:15.813950 zmfilter_1[30080].INF-zmfilter.pl [Received TERM, exiting]
2019-01-31 09:38:15.813860 zmdc[30046].INF-zmdc.pl ['zmstats.pl' sending stop to pid 30092 at 19/01/31 09:38:15]
2019-01-31 09:38:15.745830 zmfilter_2[30084].INF-zmfilter.pl [Received TERM, exiting]
2019-01-31 09:38:15.745730 zmdc[30046].INF-zmdc.pl ['zmfilter.pl --filter_id=1 --daemon' sending stop to pid 30080 at 19/01/31 09:38:15]
2019-01-31 09:38:15.681890 zmdc[30046].INF-zmdc.pl ['zmfilter.pl --filter_id=2 --daemon' sending stop to pid 30084 at 19/01/31 09:38:15]
2019-01-31 09:38:15.681774 zmc_m1[30071].INF-zm_signal.cpp/40 [Got signal 15 (Terminated), exiting]
2019-01-31 09:38:15.646020 zmdc[30046].INF-zmdc.pl ['zmc -m 1' sending stop to pid 30071 at 19/01/31 09:38:15]
2019-01-31 09:38:15.620980 zmdc[30046].INF-zmdc.pl ['zmwatch.pl' sending stop to pid 30089 at 19/01/31 09:38:15]
2019-01-31 09:38:15.596440 zmdc[30046].INF-zmdc.pl [Server exiting at 19/01/31 09:38:15]
2019-01-31 09:38:15.370020 zmpkg[30139].INF-zmpkg.pl [Command: stop]
2019-01-31 09:38:15.344360 zmpkg[30139].INF-zmpkg.pl [Sanity checking States table...]
**_2019-01-31 09:38:15.297880 zmpkg[30139].INF-zmpkg.pl [Command: stop] (command via systemd - immediately followed and successful peaceful shutdown within seconds)_**
2019-01-31 09:38:05.490451 zmc_m2[30074].INF-zm_monitor.cpp/2477 [Furnace: images:300 - Capturing at 3.03 fps, capturing bandwidth 12601bytes/sec]
2019-01-31 09:38:04.498109 zmc_m3[30077].INF-zm_monitor.cpp/2477 [Workshop: images:300 - Capturing at 3.03 fps, capturing bandwidth 9347bytes/sec]
2019-01-31 09:37:34.698316 zmc_m1[30071].INF-zm_monitor.cpp/2477 [Playroom: images:100 - Capturing at 2.08 fps, capturing bandwidth 61964bytes/sec]
2019-01-31 09:37:32.153660 zmc_m2[30074].INF-zm_monitor.cpp/2477 [Furnace: images:200 - Capturing at 2.94 fps, capturing bandwidth 12301bytes/sec]
2019-01-31 09:37:31.290389 zmc_m3[30077].INF-zm_monitor.cpp/2477 [Workshop: images:200 - Capturing at 2.94 fps, capturing bandwidth 9851bytes/sec]
2019-01-31 09:37:14.536050 zmpkg[30120].INF-zmpkg.pl [Command: stop]
2019-01-31 09:37:14.510740 zmpkg[30120].INF-zmpkg.pl [Sanity checking States table...]
**_2019-01-31 09:37:14.475850 zmpkg[30120].INF-zmpkg.pl [Command: stop] (command via web API - ignored again?)_**
2019-01-31 09:36:58.818004 zmc_m2[30074].INF-zm_monitor.cpp/2477 [Furnace: images:100 - Capturing at 3.45 fps, capturing bandwidth 14632bytes/sec]
2019-01-31 09:36:57.946429 zmc_m3[30077].INF-zm_monitor.cpp/2477 [Workshop: images:100 - Capturing at 3.57 fps, capturing bandwidth 11270bytes/sec]
2019-01-31 09:36:49.258610 zmpkg[30106].INF-zmpkg.pl [Command: restart]
2019-01-31 09:36:49.233200 zmpkg[30106].INF-zmpkg.pl [Sanity checking States table...]
**_2019-01-31 09:36:49.192080 zmpkg[30106].INF-zmpkg.pl [Command: restart] (command via web API - ignored?)_**
2019-01-31 09:36:46.172708 zmc_m1[30071].INF-zm_monitor.cpp/2477 [Playroom: images:3 - Capturing at 100.00 fps, capturing bandwidth 57433bytes/sec]
2019-01-31 09:36:45.152847 zmc_m1[30071].INF-zm_monitor.cpp/2477 [Playroom: images:1 - Capturing at 4.55 fps, capturing bandwidth 3524bytes/sec]
2019-01-31 09:36:39.890542 zmc_m1[30071].INF-zm_ffmpeg_camera.cpp/378 [Stream open rtsp://xxxxxx:yyyyyyyyy@192.168.50.50:554/videoMain, parsing streams...]
2019-01-31 09:36:39.249748 zmc_m1[30071].INF-zm_ffmpeg_camera.cpp/163 [Priming capture from rtsp://xxxxxx:yyyyyyy@192.168.50.50:554/videoMain]
2019-01-31 09:36:29.650748 zmc_m3[30077].INF-zm_monitor.cpp/2477 [Workshop: images:1 - Capturing at 16.67 fps, capturing bandwidth 3512bytes/sec]
2019-01-31 09:36:29.543028 zmc_m2[30074].INF-zm_monitor.cpp/2477 [Furnace: images:1 - Capturing at 16.67 fps, capturing bandwidth 5975bytes/sec]
2019-01-31 09:36:29.206085 zmc_m1[30071].ERR-zmc.cpp/253 [Failed to prime capture of initial monitor]
2019-01-31 09:36:29.165184 zmc_m1[30071].ERR-zm_ffmpeg_camera.cpp/602 [swscale does not support the codec format: ÿÿÿÿ]
2019-01-31 09:36:24.699438 zmc_m3[30077].INF-zm_ffmpeg_camera.cpp/378 [Stream open rtsp://xxxxxx:yyyyy@192.168.50.70:554/videoMain, parsing streams...]
2019-01-31 09:36:24.558510 zmstats[30092].INF-zmstats.pl [Stats Daemon starting in 30 seconds]
```

**Comment by zd59** [comment]
Yesterday at 13:46 GMT+1 I fetched master GIT and compiled.
Included was:
 Commits on Jan 30, 2019 - Isaac Connor fix state changing/etc

Confirmed a working state change. Before that commit I had the same problem (not working state change for about 20 days). I published two topics regarding that on a forum and confirmed now working yesterday after new commit on GIT test.
My system is Slackware64-current fresh updated.
